### ISSUE TITLE
Random crash: `Uncaught TypeError: f is null`

### ISSUE THREAD

**Issue by N-R-K** [ajbura/cinny#456]
### Describe the bug

Cinny randomly crashes.

### Reproduction

Doesn't seem to be related to any specific message/room. Couldn't figure out how to reproduce it.

### Platform and versions

```shell
1. OS GNU/Linux
2. Browser: Firefox 91.4.0
3. Cinny version: dev.cinny.in
4. Homeserver: matrix.org
```

### Additional context

Console shows the following error: `Uncaught TypeError: f is null`

**Comment by kfiven** [comment]
Without the reproduction steps, it's very hard to find such an issue. That said we fixed some crashes in 2.0.1 and 2.0.2 and hope this is fixed too.

**Comment by N-R-K** [comment]
Haven't come across any more of this. So yeah, it's probably fixed.
